### PR TITLE
Create list selectable item 

### PR DESCRIPTION
**Pull request by nypacheco**
- Add `small` prop to `List.Item`
- Add `onPress` to `List.Item` that will use `TouchableHighlight` *(from react-native)* when the prop exists.

⚠️ *waiting android screenshots*.

| android |ios |
| ----| ----    |
|listItemAndroid|listItemIos|

### CODE REVIEW

**Review by marcosricardo — APPROVED** [review]

File: packages/yoga/src/List/native/ListItem.jsx [hunk 1]
```diff
@@ -20,26 +22,55 @@ const StyledView = styled.View(
...
 ListItem.propTypes = {
+  small: PropTypes.bool,
   divided: PropTypes.bool,
+  onPress: PropTypes.func,
 };
 
 ListItem.defaultProps = {
+  small: false,
   divided: true,
+  onPress: null,
```
Comment (ggdaltoso) [inline packages/yoga/src/List/native/ListItem.jsx:71]:
```suggestion
  onPress: undefined,
```

just because 
```jsx
typeof null !== typeof(() => {})
```

**Review by evilamaior — APPROVED** [review]

File: packages/yoga/src/List/native/ListItem.jsx [hunk 2]
```diff
@@ -20,26 +22,55 @@ const StyledView = styled.View(
...
+            ${list.listItem.padding.left}px;
+        `
+  }
   `,
 );
 
-const ListItem = ({ divided, ...rest }) => (
-  <StyledView divided={divided} {...rest} />
-);
+const ListItem = ({ theme, small, divided, onPress, ...rest }) => {
```
Comment (alan-oliv) [inline packages/yoga/src/List/native/ListItem.jsx:45]: hey Yuki, we have a non-documented (sorry sorry sorry, my fault) convention that components use
we agreed to destructure the theme to the declared component level, so we avoid a lot of dot property accessors
what do you think?

**Review by alan-oliv — APPROVED** [review]
Impressive contrib! thanks for that

File: packages/yoga/src/List/native/ListItem.jsx (hunk 1 above)
Comment (nypacheco) [reply packages/yoga/src/List/native/ListItem.jsx]: Done!

File: packages/yoga/src/List/native/ListItem.jsx (hunk 2 above)
Comment (nypacheco) [reply packages/yoga/src/List/native/ListItem.jsx]: Yes, sure! Updated!

**Review by ggdaltoso — APPROVED** [review]
Beautifull! ✨

**Pull request merged**